src/zope/app/tree/tests/test_utils.py

```diff
@@ -23,12 +23,15 @@ class TestUtils(unittest.TestCase):
 
     def test_b2a_long(self):
         long_s = b'b' * 59
-        expected = 'YmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmI_'
+        expected = (
+            'YmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJiYmJi'
+            'YmJiYmJiYmJiYmI_')
```

Comment:
Personally, I would leave this on one line and add an exception for the linter.

Reply:
Fixed in https://github.com/zopefoundation/zope.app.tree/pull/9/commits/90f2305ed68c38fdc6672d069c40de8befd1227f.

Reply:
Permission to bikeshed?  `expected = 'YmJi' * 19 + 'YmI_'`

(I'm not actually serious.)

Reply:
Is 19 estimated or have you really counted the number of occurrences? :-)

Reply:
I used str.count() in a REPL. ;)